CollAction/Controllers/ProjectsController.cs

```diff
@@ -161,9 +161,15 @@ public async Task<IActionResult> Create(CreateProjectViewModel model)
             foreach (var admin in administrators)
                 _emailSender.SendEmail(admin.Email, subject, confirmationEmailAdmin);
 
-            return View("ThankYouCreate", new ThankYouCreateProjectViewModel()
+            return LocalRedirect($"~/Projects/Create/{project.Id}/{Uri.EscapeDataString(project.Name)}/thankyou");
+        }
+
+        [Authorize]
+        public IActionResult ThankYouCreate(int? id, string name)
+        {
+            return View(new ThankYouCreateProjectViewModel
```

Comment:
If we start making the name more user/url friendly, then we can't subsequently use it in the views.